app/src/main/res/values/strings.xml

```diff
@@ -576,8 +576,8 @@
         <item>144p</item>
     </string-array>
     <!-- Updates Settings -->
-    <string name="updates_setting_title">Updates</string>
-    <string name="updates_setting_description">Show a notification to prompt app update when a new version is available</string>
+    <string name="updates_setting_title">Upgrades</string>
```

Comment:
Why upgrade? In the software world it's always update not upgrade

Reply:
No, Debian always did it the right way.
If you do a refresh of the package repos, it is an update.
If you install new packages, it is an upgrade.

Reply:
That is correct. I think we had the same discussion last time. The app is not only for programmers and Linux users, but also for kids and non-techsavy people. Everyone understands the term "update", but "upgrade" can be misunderstood.

Reply:
https://english.stackexchange.com/questions/13353/are-there-any-differences-between-update-and-upgrade
So let's not be wrong?
To avoid the issue, how about "Installations" or "New versions"?

Reply:
```suggestion
    <string name="updates_setting_title">New versions</string>
```